### PR TITLE
fix: Linting

### PR DESCRIPTION
**Pull request by Major-Eth**
Fix the linting for the Icons & rename `Etherium` to `Ethereum`

### CODE REVIEW

**Review by karelianpie — APPROVED** [review]
LGTM

We should add linting to the github actions so the PR would fail if not compliant with our linting rules. What do you think @Major-Eth?

File: src/components/Icon/Arrow.tsx [hunk 1]
```diff
@@ -9,19 +9,23 @@ export const rotationDegree = {
   [IconDirection.Right]: "270",
 };
 
-const SvgUp = (props: IconDirectionProps & IconProps) => (
+export type ArrowIconProps = IconDirectionProps & IconProps;
+
+const SvgUp = (props: ArrowIconProps): React.ReactElement => (
```
Comment (karelianpie) [inline src/components/Icon/Arrow.tsx:14]: The name of this svg should probably be `SvgArrow`

Comment (karelianpie) [inline src/components/Icon/Arrow.tsx:14]: The name of this svg should probably be `SvgArrow`

File: src/components/Icon/Chevron.tsx [hunk 2]
```diff
@@ -9,21 +9,25 @@ const rotationDegree = {
   [IconDirection.Right]: "270",
 };
 
-const Svg = (props: IconDirectionProps & IconProps) => (
+export type ChevronIconProps = IconDirectionProps & IconProps;
+
+const Svg = (props: ChevronIconProps): React.ReactElement => (
```
Comment (karelianpie) [inline src/components/Icon/Chevron.tsx:14]: `SvgChevron`?

Comment (karelianpie) [inline src/components/Icon/Chevron.tsx:14]: `SvgChevron`?

File: src/components/Icon/Arrow.tsx (hunk 1 above)
Comment (Major-Eth) [reply src/components/Icon/Arrow.tsx]: Fixed by 5e05e58

Comment (Major-Eth) [reply src/components/Icon/Arrow.tsx]: Fixed by 5e05e58

File: src/components/Icon/Chevron.tsx (hunk 2 above)
Comment (Major-Eth) [reply src/components/Icon/Chevron.tsx]: Fixed by 5e05e58

**Comment by karelianpie** [comment]
@Major-Eth can we merge this please?

**Pull request merged**